modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/endpoint/ServiceRegistryEndpoint.java

```diff
@@ -717,6 +731,20 @@ public Response getOwnLoad() {
     }
   }
 
+  @GET
+  @Path("hardwareload")
+  @Produces(MediaType.TEXT_PLAIN)
```

Comment:
Same here.

Reply:
@berthob98 I think you missed this instance... :)